lib/zlib.js

```diff
@@ -786,7 +787,9 @@ function createConvenienceMethod(ctor, sync) {
 
 const kMaxBrotliParam = MathMax(...ArrayPrototypeMap(
   ObjectKeys(constants),
-  (key) => (key.startsWith('BROTLI_PARAM_') ? constants[key] : 0)
+  (key) => (StringPrototypeStartsWith(key, 'BROTLI_PARAM_') ?
+    constants[key] :
+    0)
```

Comment:
nit: I find it easier to read on one line, but you can leave it as is if you prefer.
```suggestion
  (key) =>
    (StringPrototypeStartsWith(key, 'BROTLI_PARAM_') ? constants[key] : 0)
```